src/utils/json-api.js

```diff
@@ -1,8 +1,39 @@
-function deserialize(data) {
+import Jsona from 'jsona';
+import JsonDeserializer from 'jsona/lib/builders/JsonDeserializer';
```

Comment:
Are you using this? Anyway, I don't think it's a good idea getting the function from the library itself. I would instance a JSON object and call its`deseralize` method.